doorstop/core/item.py

```diff
@@ -652,7 +653,7 @@ def _get_issues_tree(self, tree):
         if settings.REFORMAT:
             self._data['links'] = identifiers
 
-    def _get_issues_both(self, document, tree, skip):
+    def _get_issues_both(self, document, tree, skip, children):
```

Comment:
Is there any way this could be accomplished without passing `children` as a argument? This method already has a reference to the tree, so the child documents could be determined inside the `if settings.CHECK_CHILD_LINKS_STRICT:` block. I'm concerned about this breaking other functionality and showing down validation in cases were `CHECK_CHILD_LINKS_STRICT` is not needed.